monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -2425,6 +2425,20 @@ sealed abstract class Task[+A] extends Serializable {
     */
   final def void: Task[Unit] =
     this.map(_ => ())
+
+  // Deprecation directly in `Task` to avoid binary breakage just before the release
```

Comment:
I was getting `DirectMissingMethodProblem` if I put it in `Deprecations` syntax so wanted to be safe